ndlarflow/h5_to_root_ndlarflow.py

```diff
@@ -101,10 +101,17 @@ def main(argv=None):
             runID = np.array( [0], dtype='int32' )
             subrunID = np.array( [0], dtype='int32' )
             eventID = np.array( [event['id']], dtype='int32' )
-            event_start_t = np.array( [event['ts_start']], dtype='int32' )
-            event_end_t = np.array( [event['ts_end']], dtype='int32' )
+            
+            event_start_t = np.array( [event['ts_start']], dtype='float' )
```

Comment:
Is there a reason why the start and end event time types have changed from int32 to float? Should we change these back to int32? This now creates a warning during the ROOT-to-ROOT conversion step:

```
Error in <TTree::SetBranchAddress>: The pointer type given "Int_t" (3) does not correspond to the type needed "Double_t" (8) by the branch: event_start_t
Error in <TTree::SetBranchAddress>: The pointer type given "Int_t" (3) does not correspond to the type needed "Double_t" (8) by the branch: event_end_t
```

Reply:
Comment from Richie: "Due to the junk events in the open data run for 2x2, the check for 2 hits had to be placed higher in the code so Python would not complain about overrun data products for the trigger time. This was fixed by just moving the check higher in the code and I made a git commit with the times back to int32s and it runs now on open data."